api/src/main/java/org/openmrs/module/registrationcore/api/impl/RegistrationCoreProperties.java

```diff
@@ -65,4 +67,102 @@ public BiometricEngine getBiometricEngine() {
+        if (isPropertySet(propertyName)) {
+            handler = getComponentByPropertyName(propertyName, PdqErrorHandlingService.class);
+        }
+        return handler;
+    }
+    
+    /**
+     * Get the PDQ endpoint
+     * @return
+     */
+    public String getPdqEndpoint() {
+        return Context.getAdministrationService().getGlobalProperty(
+                RegistrationCoreConstants.GP_MPI_PDQ_ENDPOINT, "localhost");
+    }
+
+    /**
+     * Get the PDQ port
+     * @return
+     */
+    public Integer getPdqPort() {
+        return Integer.valueOf(Context.getAdministrationService().getGlobalProperty(
+                RegistrationCoreConstants.GP_MPI_PDQ_PORT, "8989"));
+    }
+
+    public String getPixEndpoint() {
+        return Context.getAdministrationService().getGlobalProperty(
+                RegistrationCoreConstants.GP_MPI_PIX_ENDPOINT, "localhost");
+    }
+
+    public Integer getPixPort() {
+        return Integer.valueOf(Context.getAdministrationService().getGlobalProperty(
+                RegistrationCoreConstants.GP_MPI_PIX_PORT, "3600"));
+    }
+
+    public String getMpiUsername() {
+        return Context.getAdministrationService().getGlobalProperty(
+                RegistrationCoreConstants.GP_MPI_ACCESS_USERNAME, "admin");
+    }
+
+    public String getMpiPassword() {
+        return Context.getAdministrationService().getGlobalProperty(
+                RegistrationCoreConstants.GP_MPI_ACCESS_PASSWORD, "admin");
+    }
+
+    public String getShrPatientRoot() {
+        return Context.getAdministrationService().getGlobalProperty(
+                RegistrationCoreConstants.GP_MPI_PATIENT_ROOT, "1.2.3.4.5.9");
+    }
+
+    public String getReceivingApplication() {
+        return Context.getAdministrationService().getGlobalProperty(
+                RegistrationCoreConstants.GP_MPI_RECEIVING_APPLICATION, "CR");
+    }
+
+    public String getReceivingFacility() {
+        return Context.getAdministrationService().getGlobalProperty(
+                RegistrationCoreConstants.GP_MPI_RECEIVING_FACILITY, "Clinic");
+    }
+
+    public String getSendingApplication() {
+        return Context.getAdministrationService().getGlobalProperty(
+                RegistrationCoreConstants.GP_MPI_SENDING_APPLICATION, "OPENMRS");
+    }
+
+    public String getSendingFacility() {
+        return Context.getAdministrationService().getGlobalProperty(
+                RegistrationCoreConstants.GP_MPI_SENDING_FACILITY, "LOCATION");
+    }
+
+    public String getUniversalIdType() {
+        return Context.getAdministrationService().getGlobalProperty(
+                RegistrationCoreConstants.GP_MPI_UNI_ID_TYPE, "PI");
+    }
+    
+    private boolean isPropertySet(String globalProperty) {
+        return StringUtils.isNotBlank(
+                Context.getAdministrationService().getGlobalProperty(globalProperty));
+    }
+    
+    private <T> T getComponentByPropertyName(String propertyName, Class<T> type) {
```

Comment:
I see no value in this method because it's pretty much identical to `Context.getRegisteredComponent(String, Class)`

Reply:
removed, and refactored the code that used it previously

Reply:
Did you see this comment?

Reply:
If you are referring to "I see no value in this method because it's pretty much identical to Context.getRegisteredComponent(String, Class)" yes I did see it and I acted upon it by deleting the getComponentByPropertyName method and replaced the two instances that called it with Context.getRegisteredComponent(String, Class) as I thought you were suggesting. is this not what you meant?

Reply:
Yes, but how comes this method then still exists then? You don't need this private method.

Reply:
oh as in I just did that in response to your comment, I haven't pushed the code to GitHub yet :) still on my local machine

Reply:
Makes sense